lib/src/utils/text_constants.dart

```diff
@@ -0,0 +1,10 @@
+class Texts {
+  static const appName = "Exerlog";
+
+  /// Authentication
+
+  /// Calendar Screen
+  static const startNewWorkout = "Start new workout";
```

Comment:
Did you add this with localisation in mind or just to orden the strings of the app. If localisation was the goals maybe add a method like in the appTheme where you can get the Texts.of(context) so we can check on the current localisation there in the future. You can then add that Texts.of(context) to the ThemeProvider so it provides both our texts and the theme, might have to rename the themeProvider to something else then though. If it was just for organisation ignore this.

Reply:
Just to order the strings in the app.